openreview/journal/invitation.py

```diff
@@ -6433,6 +6434,69 @@ def set_note_reviewer_message_invitation(self, note):
             signatures=[self.journal.venue_id]
         )
 
+    def set_preferred_emails_invitation(self):
+
+        venue_id = self.journal.venue_id
+
+        if openreview.tools.get_invitation(self.client, self.journal.get_preferred_emails_invitation_id()):
+            return
+
+        invitation = Invitation(
+            id=self.journal.get_preferred_emails_invitation_id(),
+            invitees=[venue_id],
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=['~Super_User1'], ## it should be the super user to get full email addresses
+            minReplies=1,
+            maxReplies=1,
+            type='Edge',
+            edit={
+                'id': {
+                    'param': {
+                        'withInvitation': self.journal.get_preferred_emails_invitation_id(),
+                        'optional': True
+                    }
+                },                
+                'ddate': {
+                    'param': {
+                        'range': [ 0, 9999999999999 ],
+                        'optional': True,
+                        'deletable': True
+                    }
+                },
+                'cdate': {
+                    'param': {
+                        'range': [ 0, 9999999999999 ],
+                        'optional': True,
+                        'deletable': True
+                    }
+                },                
```

Comment:
Why do we need to allow the `cdate` modification of the edge?

Reply:
I'm not sure, I can remove it.

Reply:
I don't understand why, but it's fine if you want to leave it.